pipelines/rh-push-to-registry-redhat-io/rh-push-to-registry-redhat-io.yaml

```diff
@@ -4,9 +4,9 @@ kind: Pipeline
 metadata:
   name: rh-push-to-registry-redhat-io
   labels:
-    app.kubernetes.io/version: "0.20.0"
+    app.kubernetes.io/version: "0.1.0"
   annotations:
-    tekton.dev/pipelines.minVersion: "0.1.0"
+    tekton.dev/pipelines.minVersion: "0.12.1"
     tekton.dev/tags: release
 spec:
   description: >-
```

Comment:
The README says it's for pushing content and here it says it's for releasing content. I'm not sure which one is better, but there's probably no reason for them to be different.